src/components/AppMenu.tsx

```diff
@@ -13,21 +13,21 @@ interface AppMenuProps extends WithLocale {
 
 export default class AppMenu extends React.Component<AppMenuProps> {
 
-    handleLanguage = () => {
-        this.props.itemSelect("language");
+    handleClose = () => {
+        this.props.itemSelect(null);
     };
 
-    handleServer = () => {
-        this.props.itemSelect("server");
+    handleLanguage = () => {
+        this.props.itemSelect('language');
     };
 
-    handleSettings = () => {
-        this.props.itemSelect("settings");
+    handleServer = () => {
+        this.props.itemSelect('server');
     };
 
-    handleClose = () => {
-        this.props.itemSelect(null);
-    };
+    // handleSettings = () => {
+    //     this.props.itemSelect("settings");
```

Comment:
```suggestion
    //     this.props.itemSelect('settings');
```

Reply:
I know this is commented out, but will be taken back into the code eventualy. Thats why it would be nice to have signle quotes already :)